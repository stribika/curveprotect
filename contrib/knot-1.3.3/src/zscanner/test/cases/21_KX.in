$ORIGIN	.
$TTL	1

; The KX is the same as the MX, so there are the differences and basics only.

; OK
@	KX	1	mail		; Relative dname
@	KX	\# 3 0001 00		; Hexadecimal rdata
@	TYPE36	\# 3 0001 00		; TYPE + Hexadecimal rdata
@	TYPE36	1	@		; TYPE
@	kx	1	@		; Type in lower-case

; KO
@	KX
