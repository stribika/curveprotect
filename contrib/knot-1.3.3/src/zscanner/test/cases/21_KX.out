OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0024
RDATA=0001 046D61696C00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0024
RDATA=0001 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0024
RDATA=0001 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0024
RDATA=0001 00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0024
RDATA=0001 00
------
WARNG=ZSCANNER_EBAD_NUMBER
------
